src/vivarium_public_health/mslt/delay.py

```diff
@@ -477,3 +477,18 @@ def incidence_adjustment(self, disease, index, incidence_rate):
         pif = (mean_bau_rr - mean_int_rr) / mean_bau_rr
         pif = pif.fillna(0.0)
         return incidence_rate * (1 - pif)
+
+def pivot_load(builder, entity_key):
+    """Helper method for loading dataframe from artifact.
+
+    Performs a long to wide conversion if dataframe has an index column
+    named 'measure'.
+
+    """
+    data = builder.data.load(entity_key)
+
+    if 'measure' in data.columns :
+        data  = data.pivot_table(index = [i for i in data.columns if i not in ['measure','value']], columns = 'measure', \
```

Comment:
Pivot table is actually quite slow, if I recall.  You might get better performance using dataframe.unstack() and then fixing up column/index names.  

Style nitpicks: when you're giving keyword args, no spaces around the '=' operator.  You also should stay away from explicit line continuation as recommended by pep8 (https://www.python.org/dev/peps/pep-0008/#maximum-line-length)

When doing method chaining in pandas, I find the following format easiest to read:
```
index = [i for i in data.columns if i not in ['measure','value']]
data = (
    data.pivot_table(index=index, columns='measure', values='value)
    .rename_axis(None, axis=1)
    .reset_index()
)
```
This is especially helpful if your method chains get quite long.  The most important code optimization is for readability.